tests/integration_tests/functional/test_api.py

```diff
@@ -389,8 +389,7 @@ def test_api_machine_config(test_microvm_with_api):
     test_microvm.api.machine_config.patch(mem_size_mib=bad_size)
 
     fail_msg = re.escape(
-        "Invalid Memory Configuration: MmapRegion(Mmap(Os { code: "
-        "12, kind: OutOfMemory, message: Out of memory }))"
+        "Invalid Memory Configuration: MemefdSetLen(Custom { kind: InvalidInput, error: TryFromIntError(()) })"
```

Comment:
same typo as before s/Memefd/Memfd.
I actually think this change may go to the same commit where memfd was added, but no strong preference.

Reply:
fixed